Brixcer_Box/src/main.cpp

```diff
@@ -30,13 +30,18 @@
 #define Button1 D16
 #define LedChain D10
 
+
```

Comment:
Is this a sniffing thing? Why was this added?

Reply:
To separate out button definitions from other globals

Reply:
okay sounds good. we should set up some sniffs. Not sure what they use in C++ though.